recipes/automake/all/test_package/conanfile.py

```diff
@@ -1,89 +1,57 @@
-from conans import AutoToolsBuildEnvironment, ConanFile, tools
-from conan.tools.microsoft import is_msvc
-from contextlib import contextmanager
-import os
 import shutil
 
-required_conan_version = ">=1.45.0"
+from conan import ConanFile
+from conan.tools.build import can_run
+from conan.tools.gnu import Autotools
+from conan.tools.layout import basic_layout
+from conan.tools.microsoft import unix_path
+
+required_conan_version = ">=1.50.0"
```

Comment:
```suggestion
required_conan_version = ">=1.51.3"
```

I saw your comment about the test\_package and the package versions drifting apart over time anyway. Just thought I would post this anyway in case.